From 67642a6bbf7261626f41b84cccf9b55b93c4cbc9 Mon Sep 17 00:00:00 2001
From: Anatol Belski <anbelski@linux.microsoft.com>
Date: Thu, 14 Jan 2021 17:36:23 +0000
Subject: [PATCH] xmlsec1: Fix configure QA error caused by host lookup path

ERROR: mc:my-sdk:xmlsec1-1.2.30-r0 do_configure: QA Issue: This autoconf log indicates errors, it looked at host include and/or library paths while determining system capabilities.

It will eventually arise after the configure QA as the configure script should only look at the staging sysroot dir, not at the host.

Upstream-Status: Inappropriate [embedded specific]
Signed-off-by: Anatol Belski <anbelski@linux.microsoft.com>
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index b8b0542d..795355af 100644
--- a/configure.ac
+++ b/configure.ac
@@ -306,8 +306,8 @@ fi
 dnl ==========================================================================
 dnl Common installation locations
 dnl ==========================================================================
-COMMON_INCLUDE_DIR="/usr/include /usr/local/include"
-COMMON_LIB_DIR="/usr/lib /usr/lib64 /usr/local/lib"
+COMMON_INCLUDE_DIR="${STAGING_INCDIR}"
+COMMON_LIB_DIR="${STAGING_LIBDIR}"
 case $host in
     i*86-*-linux-gnu)   COMMON_LIB_DIR="$COMMON_LIB_DIR /usr/lib/i386-linux-gnu" ;;
     x86_64-*-linux-gnu) COMMON_LIB_DIR="$COMMON_LIB_DIR /usr/lib/x86_64-linux-gnu" ;;
